hvplot/__init__.py

```diff
@@ -213,8 +237,7 @@ def help(kind=None, docstring=True, generic=True, style=True):
 
 
 def post_patch(extension='bokeh', logo=False):
-    if extension and not getattr(_hv.extension, '_loaded', False):
-        hvplot_extension(extension, logo=logo)
+    hvplot_extension(extension, logo=logo)
```

Comment:
We could have an argument that restores the old behavior, but I don't have a strong opinion.